src/VCR/Configuration.php

```diff
@@ -14,6 +14,11 @@
  */
 class Configuration
 {
+    public function __construct() {
+        $this->matchObserver = new MatchObserver();
```

Comment:
I did not pay careful enough attention to the factory. Cool, I will move logic to there.